From: Yifeng <wuyifeng@nd.com.cn>
Date: Tue, 13 Dec 2022 07:40:00 +0000
Subject: Disable fingerprinting

Fix version: 1.2.1
When we enable the fingerprinting, some websites will not be loaded successfully, for example,
https://phet.colorado.edu/sims/html/vector-addition/latest/vector-addition_en.html,
so we have to disable it.
Relate to AP9-11018.
Relate to Multiple-fingerprinting-mitigations.patch.

---
 content/child/runtime_features.cc | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/content/child/runtime_features.cc b/content/child/runtime_features.cc
--- a/content/child/runtime_features.cc
+++ b/content/child/runtime_features.cc
@@ -531,12 +531,9 @@ void SetCustomizedRuntimeFeaturesFromCombinedArgs(
   if (!command_line.HasSwitch(switches::kDisableYUVImageDecoding) &&
       base::FeatureList::IsEnabled(
           blink::features::kDecodeLossyWebPImagesToYUV)) {
-  WebRuntimeFeatures::EnableFingerprintingClientRectsNoise(
-      !command_line.HasSwitch(switches::kFingerprintingClientRectsNoise));
-  WebRuntimeFeatures::EnableFingerprintingCanvasMeasureTextNoise(
-      !command_line.HasSwitch(switches::kFingerprintingCanvasMeasureTextNoise));
-  WebRuntimeFeatures::EnableFingerprintingCanvasImageDataNoise(
-      !command_line.HasSwitch(switches::kFingerprintingCanvasImageDataNoise));
+  WebRuntimeFeatures::EnableFingerprintingClientRectsNoise(false);
+  WebRuntimeFeatures::EnableFingerprintingCanvasMeasureTextNoise(false);
+  WebRuntimeFeatures::EnableFingerprintingCanvasImageDataNoise(false);

     WebRuntimeFeatures::EnableDecodeLossyWebPImagesToYUV(true);
   }
--